tests/lti/models/auth_data_test.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
```

Comment:
We're putting these `coding` and `unicode_literals` lines at the top of every Python file